### PR TITLE
Dmvp 798: Setup Gitlab Runner in k8s and add semantic-release, pre-commit and prevent wrong commit-msg

### PR DESCRIPTION
**Pull request by 0katrinpetrosyan0**

### CODE REVIEW

File: modules/gitlab-runner/main.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+resource "kubectl_manifest" "values_yaml" {
+  count     = length(data.kubectl_path_documents.values.documents)
+  yaml_body = element(data.kubectl_path_documents.values.documents, count.index)
+}
+
+resource "helm_release" "gitlab-runner" {
+  name      = "gitlab-runner-chart"
+  namespace = "gitlab-runner"
```
Comment (mrdntgrn) [inline modules/gitlab-runner/main.tf:16]: the name and namespace should be configurable variables

File: modules/gitlab-runner/main.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+data "kubectl_path_documents" "values" {
```
Comment (mrdntgrn) [inline modules/gitlab-runner/main.tf:1]: why we need kubectl, seems helm doing the job

Comment (0katrinpetrosyan0) [reply modules/gitlab-runner/main.tf:1]: that is for adding our yaml files

File: modules/gitlab-runner/README.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
+# SETUP gitlab-runner in k8s
+
+## Provider configuration
+
+```
+terraform {
```
Comment (mrdntgrn) [inline modules/gitlab-runner/README.md:6]: please have this block included in module in new versions.tf file

Comment (0katrinpetrosyan0) [reply modules/gitlab-runner/README.md]: okay I add this file

**Pull request merged**